### DISCUSSION TITLE
Background gene list for enrichment

### DISCUSSION THREAD

**Discussion by fengx90** [egeulgen/pathfindR#213] [Q&A]
Dear Ege,

As I understand that PathfindR uses all the genes of a selected species (e.g. mouse or human) as background for the functional enrichment, is this correct? 
As the backgrounds affect the statistical analysis, I wonder if there is a way to use a customized gene/protein list as background? Thanks very much!

Best,
Feng

**Comment by ozanozisik** [comment]
Hello,
By default, the genes in the protein interaction network are used as background. Currently there is not a straightforward way to change the background gene set. We will consider this for the following updates. Thank you.
Best,
Ozan

**Reply by fengx90** [reply]
Hi Ozan,

Thanks for the fast reply. 
Just to clarify: it is not the whole genome of a species. So if I use GO-BP set for the enrichment, the background will be all genes that have annotations in BP, right? 
Thanks for considering it for future updates.

Best,
Feng

**Reply by ozanozisik** [reply]
Hello,
As we select the genes from the PPI, background is composed of the genes in the PPI, not the genes in the annotation database (e.g. genes annotated in GO:BP). A second filtering could be considered to keep only the ones in the annotation database as background, currently we do not do this.
Best,
Ozan
